New Entry: General file format (#730)
Codecademy/docs: content/general/concepts/file-formats/file-formats.md

---

> Title: 'File formats'

Reviewer: ```suggestion
Title: 'File Formats'
```

"Title Case"

> Description: 'File formats'

Author: ```suggestion
Description: 'File formats determine how information or data, such as text and images, are created, stored, and read.'
```

Subjects:
  - 'Computer Science'
  - 'Data Science'
Tags:

>   - 'Files'

Reviewer: ```suggestion
  - 'Files'
  - 'Types'
```

CatalogContent:

>   - 'paths/code-foundations'

Reviewer: Hmmm I'm having a hard time finding a suitable `learn-` link for this one 😂

Author: 😆 what do you think about `learn-the-command-line`?

Reviewer: Works for me 😃

Author: ```suggestion
  - 'learn-the-command-line'
  - 'paths/code-foundations'
```

  - 'paths/computer-science'
---

File formats determine how information or data, such as text and images, are created, stored, and read. Each file format has a unique extension. For example, a CSV (Comma Separated Values) file has the extension **.csv**.

> Certain formats are designed to compress data for fast processing. Others may contain more features, like how PDF (Portable Document Format) files can contain graphs and charts along with table data whereas a CSV file can only contain plain table data.

Author: ```suggestion
Certain formats are designed to compress data for fast processing. Others may contain more features, like how XLSX (Microsoft Excel) files can contain graphs and charts along with table data whereas a CSV file can only contain plain table data.
```

Files can also be converted to other formats, however, some features for data display may be lost depending on the type of conversion.

> ## Types of file formats

Reviewer: Aren't headers for Docs entries just "Title Case" outright?

Author: ```suggestion
## Types of File Formats
```

File formats exist for different types of files. Some common file types and formats are listed below:

| File Format                                                                             | Extension | Description                                                                                                  |

> | --------------------------------------------------------------------------------------- | --------- | ------------------------------------------------------------------------------------------------------------ |

Reviewer: ```suggestion
| :---------------------------------------------------------------------------------------: | :---------: | ------------------------------------------------------------------------------------------------------------ |
```
